server/app/services/mail-service.js

```diff
@@ -184,11 +185,57 @@ async function sendWelcomeMail(email) {
     await transporter.sendMail(mailOptions);
 }
 
+/**
+ * Send a contact form mail
+ *
+ * @param formName {String} french contact form name
+ * @param emails {String[]} recipient email address
+ * @param value {Object} form value
+ * @returns {Promise<void>} Nothing
+ */
+async function sendContactForm(formName, emails, values) {
```

Comment:
Maybe you have a better idea to implement formName parameter ? I'm not very proud of this implementation.